npm-shrinkwrap.json

```diff
@@ -75,179 +77,186 @@
     },
     "closure-linter-wrapper": {
       "version": "0.2.9",
-      "from": "closure-linter-wrapper@~0.2.0",
+      "from": "https://registry.npmjs.org/closure-linter-wrapper/-/closure-linter-wrapper-0.2.9.tgz",
       "resolved": "https://registry.npmjs.org/closure-linter-wrapper/-/closure-linter-wrapper-0.2.9.tgz",
       "dependencies": {
         "colors": {
           "version": "0.6.2",
-          "from": "colors@~0.6.2",
+          "from": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz",
           "resolved": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz"
         }
       }
     },
     "express": {
       "version": "4.10.7",
-      "from": "express@^4.7.2",
+      "from": "https://registry.npmjs.org/express/-/express-4.10.7.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.